modules/test/playwright/tests/export-import-web/pages/CompanyExportImportPage.ts

```diff
@@ -31,9 +32,14 @@ export class CompanyExportImportPage {
 
 		await this.page.getByLabel(itemLabel).click();
 
-		const exportName = 'MyExport-' + getRandomString();
+		let exportName = 'MyExport-' + getRandomString();
```

Comment:
You are calling the getRandomString function even if the defaultTaskName is true. That is weird